include/picongpu/unitless/simulation.unitless

```diff
@@ -188,13 +197,14 @@ namespace picongpu
             constexpr float_64 typicalNumParticlesPerMacroParticle() const
             {
                 return (si.getBaseDensity() * si.getCellSize().productOfComponents())
-                    / float_64(getTypicalNumParticlesPerCell());
+                    / getTypicalNumParticlesPerCell<float_64>();
             }
         };
```

Comment:
Why's this hard-coded to `float_64`?

Reply:
To avoid compiler warnings, we cast from an integer type to floating point type. It was an explicit cast before too.